Detector/DetFCChhBaseline1/compact/FCChh_DectMaster.xml

```diff
@@ -31,7 +31,7 @@
   <include ref="./FCChh_Shielding.xml" />
 
 
-  <include ref="../../DetFCChhTrackerSimple/compact/Tracker.xml" />
+  <include ref="../../DetFCChhTrackerTkLayout/compact/Tracker.xml" />
```

Comment:
Is the "Simple" tracker actually still needed? As far as I understand it has overlaps as it stands? If that's the case, I think we should either remove it or fix the overlaps.